views/web.py

```diff
@@ -286,6 +280,7 @@ def love():
             flash('Secret love sent to {}!'.format(real_display_str))
             return redirect(url_for('home'))
         else:
+            flash('Love sent to {}!'.format(real_display_str))
             hash_key = link_id if link_id else create_love_link(real_display_str, message).hash_key
             return redirect(url_for('sent', message=message, recipients=real_display_str, link_id=hash_key))
 
```

Comment:
the exception below still isn't being handled with a proper redirect. You should have `return redirect(url_for('home'))` at the bottom of the caught exception. Otherwise the user gets a 500 and these flashes don't go anywhere.